game/src/enemy/core/Enemy.gd

```diff
@@ -201,6 +202,30 @@ func _on_body_entering_vitals(body: Node) -> void:
 		body.inflict_damage(self)
 
 
+func _drop_item(percentage: int):
+	var rng = RandomNumberGenerator.new()
+	rng.randomize()
+	var roll = rng.randi_range(0, 100)
+	print(roll)
```

Comment:
you gotta rm your prints

Reply:
Moved to next review: https://github.com/quantum-defence/quantum-defence/pull/103#discussion_r673817651